utils/connect.js

```diff
@@ -1,6 +1,33 @@
-const { connect: nearConnect } = require('near-api-js');
+const { 
+    connect: nearConnect,
+    Account,
+    multisig: {
+        MULTISIG_CHANGE_METHODS
+    }
+} = require('near-api-js');
+const { AccountMultisig } = require('./2fa');
 
 module.exports = async function connect({ keyStore, ...options }) {
-    // TODO: Avoid need to wrap in deps
-    return await nearConnect({ ...options, deps: { keyStore }});
+    const near = await nearConnect({ ...options, deps: { keyStore }});
+    near.account = async (accountId) => {
+        const account = new Account(near.connection, accountId);
+        await account.state();
+        const accessKeys = await account.getAccessKeys();
+        const hasFAK = accessKeys.find((k) => k.access_key.permission && k.access_key.permission === 'FullAccess');
+        if (hasFAK) {
+            return account;
+        }
+        
+        const use2fa = accessKeys.find((k) => 
+            k.access_key.permission && 
+            k.access_key.permission.FunctionCall &&
+            k.access_key.permission.FunctionCall.method_names.some((mn) => MULTISIG_CHANGE_METHODS.includes(mn))
+        );
+        if (use2fa) {
+            return new AccountMultisig(near.connection, accountId);
+        }
+
+        throw Error('No account matching', accountId);
```

Comment:
This error message would make me think that account doesn't exist at all, which isn't true. Also `accountId` is basically passed as `fileName` parameter of `Error` constructor, which is probably unintentional. 

I think we should do something more like:

```
throw new Error(`Account ${accountId} doesn't have either full access keys or multisig`);
```

however I think even then we also need to support case where there is local limited access key as you can use it with `near call` or `near repl`